tardis/plasma/standard_plasmas.py

```diff
@@ -1,26 +1,14 @@
 import logging
 
 from tardis.plasma import BasePlasma
-from tardis.plasma.properties.plasma_input import (TRadiative, AtomicData, Abundance,
-                                        Density, TimeExplosion)
-from tardis.plasma.properties import (
-    BetaRadiation, LevelBoltzmannFactor, Levels, Lines,
-    SelectedAtoms, AtomicMass, LTEPartitionFunction, LevelPopulationLTE,
-    LevelNumberDensity, PhiSahaLTE, GElectron, IonizationData, NumberDensity,
-    IonNumberDensity, LinesLowerLevelIndex, LinesUpperLevelIndex, TauSobolev)
+from tardis.plasma.properties.property_collections import LTEInputs
 
 logger = logging.getLogger(__name__)
 
 class LTEPlasma(BasePlasma):
 
     def __init__(self, t_rad, abundance, density, time_explosion, atomic_data):
-        plasma_modules = [TRadiative, BetaRadiation, LevelBoltzmannFactor,
-                          Levels, Lines, AtomicData, Abundance, SelectedAtoms,
-                          AtomicMass, LTEPartitionFunction, LevelPopulationLTE,
-                          PhiSahaLTE, GElectron, IonizationData, Density,
-                          NumberDensity, IonNumberDensity, LevelNumberDensity,
-                          LinesLowerLevelIndex, LinesUpperLevelIndex,
-                          TauSobolev, TimeExplosion]
+        plasma_modules = LTEInputs().list
```

Comment:
I think this is allright. But I would use it as a list. I think that `LTEInputs` should also only contain the inputs not every lte plasma module.